<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StructurePage</class>
 <widget class="QWizardPage" name="StructurePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WizardPage</string>
  </property>
  <property name="title">
   <string>New Project</string>
  </property>
  <property name="subTitle">
   <string>Choose your story structure. You will be able modify it at your leisure afterward.</string>
  </property>
  <attribute name="pageId">
   <string notr="true">3</string>
  </attribute>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QListWidget" name="bookTemplatesListWidget"/>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="1">
      <widget class="QSpinBox" name="bookCountSpinBox">
       <property name="accelerated">
        <bool>true</bool>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>999</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Books :</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Chapters :</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Acts :</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Scenes :</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="3" column="1">
      <widget class="QSpinBox" name="actCountSpinBox"/>
     </item>
     <item row="4" column="1">
      <widget class="QSpinBox" name="chapterCountSpinBox"/>
     </item>
     <item row="5" column="1">
      <widget class="QSpinBox" name="sceneCountSpinBox"/>
     </item>
     <item row="6" column="1">
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Count :</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTreeWidget" name="bookPreviewTreeWidget">
     <property name="verticalScrollMode">
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
     <property name="animated">
      <bool>true</bool>
     </property>
     <property name="headerHidden">
      <bool>true</bool>
     </property>
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>bookTemplatesListWidget</tabstop>
  <tabstop>bookCountSpinBox</tabstop>
  <tabstop>actCountSpinBox</tabstop>
  <tabstop>chapterCountSpinBox</tabstop>
  <tabstop>sceneCountSpinBox</tabstop>
  <tabstop>bookPreviewTreeWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
